src/App.vue

```diff
@@ -627,13 +630,11 @@ export default class App extends Mixins(
     })
 
     // fetch draft filing
-    let draftFiling = await this.fetchDraftDissolution()
+    let draftFiling = await this.fetchDraftDissolution().catch(error => { throw error })
```

Comment:
Re-throw the error explicitly, instead of this method throwing the error by default.

Also we no longer need to check whether `draftFiling` exists since this code is not run if the fetch failed.

Same change below for incorporation filing.